crates/sui-graphql-rpc/src/types/transaction_block.rs

```diff
@@ -50,28 +76,38 @@ impl From<SuiTransactionBlockResponse> for TransactionBlock {
             bcs: Some(Base64::from(&tx_block.raw_transaction)),
             gas_input,
             epoch_id: None,
-            kind: None, // TODO (stefan) fix this
+            // kind: None, // TODO (stefan) fix this
         }
     }
 }
 
-#[ComplexObject]
-impl TransactionBlock {
-    async fn digest(&self) -> String {
-        self.digest.to_string()
-    }
-
-    async fn expiration(&self, ctx: &Context<'_>) -> Result<Option<Epoch>> {
-        match self.epoch_id {
-            None => Ok(None),
-            Some(epoch_id) => {
-                let epoch = ctx
-                    .data_unchecked::<PgManager>()
-                    .fetch_epoch_strict(epoch_id)
-                    .await
-                    .extend()?;
-                Ok(Some(epoch))
+// This is required for the sdk reference implementations
+// TODO remove this once the full DB implementation is complete
+impl From<&SuiTransactionBlockEffects> for TransactionBlockEffects {
```

Comment:
same here, p sure most of the code exists on main as well?

Reply:
I rebased, I think, so not sure.